File: backend/src/main/java/capstone/rt04/retailbackend/request/staff/StaffDetailsUpdateRequest.java
package capstone.rt04.retailbackend.request.staff;
import capstone.rt04.retailbackend.entities.Department;
import capstone.rt04.retailbackend.entities.Staff;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.validation.constraints.NotNull;



import capstone.rt04.retailbackend.entities.*;
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class StaffDetailsUpdateRequest {
    private Staff staff;
    private Long roleId;
    private Long departmentId;
    private Address address;
    private Long storeId;

}

File: frontend/staff-web-app/src/models/staff/StaffDetailsUpdateRequest.js
export default class StaffDetailsUpdateRequest {
    staff;
    address;
    roleId;
    departmentId;
    storeId;

    constructor(staff, roleId, departmentId, address, storeId) {
        this.staff = staff;
        this.address = address;
        this.roleId = roleId;
        this.departmentId = departmentId;
        this.storeId = storeId;
    }
}
